src/misc/lv_anim.c

```diff
@@ -550,3 +600,75 @@ static bool remove_concurrent_anims(lv_anim_t * a_current)
 
     return del_any;
 }
+
+static void anim_set_start_values(lv_anim_t * a)
+{
+    a->run_round = state.anim_run_round;
+    a->last_timer_run = lv_tick_get();
+
+    if(a->early_apply) {
+        if(a->get_value_cb) {
+            int32_t v_ofs = a->get_value_cb(a);
+            a->start_value += v_ofs;
+            a->end_value += v_ofs;
+
+        }
+
+        resolve_time(a);
+
+        /*Do not let two animations for the same 'var' with the same 'exec_cb'*/
```

Comment:
Please, why we can't have two animations with same var to same exec callback?

Reply:
It would lead to concurrency (who got registered to the list first) and undefined behavior.

Imagine 2 animations setting the y offset of the same ball with a time delay between them. Which one is right? Where the ball should really be?